### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by milesgranger**

### CODE REVIEW

File: docs/components/machine/machine.rst [hunk 1]
```diff
@@ -6,26 +6,64 @@ a model, dataset, metadata and everything
...
+                - sklearn.preprocessing.MinMaxScaler
+                - gordo.machine.model.models.KerasAutoEncoder:
+                    kind: feedforward_hourglass
+
+And to construct this into a python object:
+
+.. code-block:: python
+
+    >>> from gordo.machine import Machine
+    >>> machine = Machine.from_config(config, project_name='test-proj')
```
Comment (epa095) [inline docs/components/machine/machine.rst:39]: config is here the result of yaml.parse?

Comment (epa095) [reply docs/components/machine/machine.rst:39]: (i.e. maybe have a that in a side-sentence or something)

File: docs/components/machine/metadata.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Metadata
+--------
+
+Each Machine is entitled to have Metadata, which can be set at the Machine.metadata
```
Comment (epa095) [inline docs/components/machine/metadata.rst:4]: damn, entitled!

File: docs/components/serializer.rst [hunk 3]
```diff
@@ -1,23 +1,39 @@
...
     :show-inheritance:
 
-Pipeline into definition
-========================
+
+Into Definitiion
+================
+
+The ability to take a Python object, such as a scikit-learn
+pipeline and convert it into a primitive ``dict``, and thus inserted
```
Comment (epa095) [inline docs/components/serializer.rst]: "and thus" -> "which can then be" or something like that

File: gordo/machine/machine.py [hunk 4]
```diff
@@ -75,6 +75,23 @@ def __init__(
     def from_config(  # type: ignore
         cls, config: Dict[str, Any], project_name: str, config_globals=None
     ):
+        """
+        Construct an instance from a block of YAML config file which represents
```
Comment (epa095) [inline gordo/machine/machine.py:79]: not from a yaml really, but from a dict which can come from the yaml

File: docs/components/machine/metadata.rst (hunk 2 above)
Comment (milesgranger) [reply docs/components/machine/metadata.rst:4]: that's socialism my man.

**Review by epa095 — APPROVED** [review]

**Comment by epa095** [comment]
Minor comments really. I trust you to fix as you see fit and merge at will.

File: gordo/machine/machine.py (hunk 4 above)
Comment (milesgranger) [reply gordo/machine/machine.py:79]: appended a statement at the end, that it is to loaded in as a dict. :+1:

**Pull request merged**